tests/nodescaling/README.md

```diff
@@ -80,9 +82,78 @@ terratest:
         quantity: 1
   ```
 
+### AKS
+
+```yaml
+terratest:
+  kubernetesVersion: "1.28.5"
```

Comment:
supernit; I hate that we hard code versions in our readme, because it inevitably gets out of date. its just annoying

Reply:
I mean I can remove it, not a big deal either way to me

Reply:
Resolved